twitter-dm-notifier/index.ts

```diff
@@ -0,0 +1,113 @@
+import Twitter from 'twitter';
+import dotenv from 'dotenv';
+import moment, { Moment } from 'moment';
+import type { MessageCreateEvent, User } from '../lib/twitter';
+import type { SlackInterface } from '../lib/slack';
+dotenv.config();
+
+const twitterClient = new Twitter({
+    consumer_key: process.env.TWITTER_CONSUMER_KEY!,
+    consumer_secret: process.env.TWITTER_CONSUMER_SECRET!,
+    access_token_key: process.env.TWITTER_ACCESS_TOKEN_KEY!,
+    access_token_secret: process.env.TWITTER_ACCESS_TOKEN_SECRET!,
+});
+
+const getDMs = async (options: { after?: string; count: number; } = { count: 50 }) => {
+    const { after, count } = options;
```

Comment:
[INFO]

```suggestion
const getDMs = async ({ after, count }: { after?: string; count: number; } = { count: 50 }) => {
```

とも書けます

Reply:
うーん別によくない? よくないか? と思ったけど、まあ、そうします